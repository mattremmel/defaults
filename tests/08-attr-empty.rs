use defaults::Defaults;

#[derive(Defaults)]
struct Foo {
    #[default]
    x: usize,
    y: usize,
}

fn main() {}
