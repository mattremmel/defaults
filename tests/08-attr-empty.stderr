error: expected attribute arguments in parentheses: #[default(...)]
 --> $DIR/08-attr-empty.rs:5:5
  |
5 |     #[default]
  |     ^^^^^^^^^^
